content/vendor/replicated-helm/using-helm-charts.md

```diff
@@ -1,15 +1,15 @@
 ---
 date: 2019-12-27
-linktitle: "Using Helm Charts"
-title: Using Helm Charts
+linktitle: "Using Replicated Helm Charts"
+title: Using Replicated Helm Charts
 aliases:
-  - /vendor/helm/
+  - /vendor/replicated-helm/
 weight: 20500
 ---
 
-Helm is a popular package manager for Kubernetes applications. 
-Replicated KOTS supports delivering Helm charts as an enterprise application, or including Helm charts as components of an application. 
-A KOTS application can support more than one Helm chart, and can support more than a single instance of any Helm chart.
+Helm is a popular package manager for Kubernetes applications. Replicated KOTS supports delivering Helm charts as an enterprise application, or including Helm charts as components of an application. A KOTS application can support more than one Helm chart, and can support more than a single instance of any Helm chart.
+
+> The Replicated Helm workflow uses KOTS to render and deploy Helm charts. For new applications, the [native Helm workflow](/vendor/native-helm/) is the preferred method because it supports more features of Helm, such as hooks and weights.
```

Comment:
s/ The Replicated Helm workflow / The Replicated deployment workflow of Helm

s/ the [native Helm workflow] / the [native Helm deployment workflow]